### ISSUE TITLE
[Client] Latest binaries on release page seg fault instantl

### ISSUE THREAD

**Issue by rminderhoud** [dev-osrose/osIROSE-new#147]
Using the latest `trose.exe` et al. from the release page with `129_129en` causes the client to instantly seg fault.

```
C:\dev\rose-129-129en>trose.exe @TRIGGER_SOFT@ 192.168.0.104
Segmentation fault
```

**Comment by L3nn0x** [comment]
I think that's because it's not the latest. You'll want to ask raven for it

**Comment by RavenX8** [comment]
You're missing _server from that command. Please try again with _server in the command
`trose.exe @TRIGGER_SOFT@ _server 192.168.0.104`

EDIT: please check the current directory for a core dump file and attach it.

**Comment by rminderhoud** [comment]
[TRose 2018-11-28 02-44-58.zip](https://github.com/dev-osrose/osIROSE-new/files/2622731/TRose.2018-11-28.02-44-58.zip)

**Comment by RavenX8** [comment]
DId using the 112-112en client fix your issue? 129-129en is a evo client and won't work with our trose.exe

**Comment by rminderhoud** [comment]
Confirming client 112_112na works.

**Comment by rminderhoud** [comment]
Maybe docs should be updated?

**Comment by RavenX8** [comment]
hm which doc? the readme?

**Comment by L3nn0x** [comment]
Probably

**Comment by rminderhoud** [comment]
Closed by #174
